scripts/create-package/plop-templates-react/package.json.hbs

```diff
@@ -44,10 +45,14 @@
     "react-test-renderer": ""
   },
   "dependencies": {
-    "@fluentui/theme": "",
     "@fluentui/react-compose": "",
+    "@fluentui/react-hooks": "",
+    "@fluentui/react-make-styles": "",
+    "@fluentui/react-theme": "",
     "@fluentui/react-theme-provider": "",
+    "@fluentui/react-utils": "",
     "@fluentui/set-version": "",
+    "@testing-library/react-hooks": "",
```

Comment:
Whoops this needs to go in devdependencies, I think it's my bad for having them it there in the first place

Reply:
I've made that change too in `react-menu`

Reply:
Ok. will update!